### ISSUE TITLE
Info icon present in action panel is not accessible through keyboard

### ISSUE THREAD

**Issue by Xiaoyu-Huang** [Azure/LogicAppsUX#4179]
### Describe the issue 

Info icon present in action panel is not accessible through keyboard. it is accessible by hovering over the icon.

![image](<https://github.com/Azure/LogicAppsUX/assets/22843713/7fb221a3-fc46-4f1d-b3ae-ea4155999aad>)

### Are you using new designer or old designer 

New Designer (With action panels)

### Screenshots or Videos 

*No response*

### Browser 

Edge

### Additional context 

*No response*
